miio/airconditioningcompanion.py

```diff
@@ -250,7 +269,17 @@ def __json__(self):
 
 
 class AirConditioningCompanion(Device):
-    """Main class representing Xiaomi Air Conditioning Companion."""
+    """Main class representing Xiaomi Air Conditioning Companion V1 and V2."""
+
+    def __init__(self, ip: str = None, token: str = None, start_id: int = 0,
+                 debug: int = 0, lazy_discover: bool = True,
+                 model: str = MODEL_ACPARTNER_V2) -> None:
+        super().__init__(ip, token, start_id, debug, lazy_discover)
+
+        if model in MODELS_SUPPORTED:
+            self.model = model
+        else:
+            self.model = MODEL_ACPARTNER_V2
```

Comment:
This defaults to v2 if an unknown model is given? Maybe add a warning logging for that case.